youtube_dl/extractor/fc2.py

```diff
@@ -99,25 +98,22 @@ def _real_extract(self, url):
         info_webpage = self._download_webpage(
             info_url, video_id, note='Downloading info page')
         info = compat_urlparse.parse_qs(info_webpage)
-
-        if 'err_code' in info:
-            # most of the time we can still download wideo even if err_code is 403 or 602
-            self.report_warning(
-                'Error code was: %s... but still trying' % info['err_code'][0])
-
-        if 'filepath' not in info:
-            raise ExtractorError('Cannot download file. Are you logged in?')
-
-        video_url = info['filepath'][0] + '?mid=' + info['mid'][0]
         title_info = info.get('title')
         if title_info:
             title = title_info[0]
 
+        video_info_url = 'https://video.fc2.com/api/v3/videoplaylist/{}?sh=1&fs=0'.format(video_id)
```

Comment:
1. No `{}`.
2. Inline.
3. Query to `query`.

Reply:
I am confused about 3. Query to `query`